src/game/obj_behaviors_2.c

```diff
@@ -47,18 +47,15 @@
 #include "spawn_sound.h"
 #include "puppylights.h"
 
-#define POS_OP_SAVE_POSITION 0
-#define POS_OP_COMPUTE_VELOCITY 1
-#define POS_OP_RESTORE_POSITION 2
+//! TODO: remove static
 
-#define o gCurrentObject
```

Comment:
exported from somewhere else? helps text editors with autocomplete if this is defined

Reply:
same as above